### PR TITLE
Oj 865 create browser tests for translations

### PR DESCRIPTION
**Pull request by MattBidewell**
### What changed

Browser test to test that with the cookie change and a reload - the content is translated from English to Welsh. The tests are set to skip for now.

### Why did it change

Small content change to check that the search page is translated when the cookie is set to welsh. The content is not finalised and is just a google translate for now. The real content will be replacing the current content.

### Issue tracking

- [OJ-865](https://govukverify.atlassian.net/browse/OJ-865)

### Environment variables or secrets

- [ ] No environment variables or secrets were added or changed

- [ ] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

### CODE REVIEW

**Comment by MattBidewell** [comment]
Dont merge yet -skip isnt working correctly.

**Review by gokhakan — APPROVED** [review]

**Comment by gokhakan** [comment]
Very neat.

File: test/browser/step_definitions/search.js [hunk 1]
```diff
@@ -27,3 +27,15 @@ Then("they should see the previous address search page", function () {
   const searchPage = new SearchPage(this.page);
   expect(searchPage.isCurrentPage()).to.be.true;
 });
+
+Then("they should see the search page content in Welsh", async function () {
+  const searchPage = new SearchPage(this.page);
+  expect(await searchPage.getPageTitle()).to.include(
+    "Dewch o hyd i'ch cyfeiriad"
```
Comment (MattBidewell) [inline test/browser/step_definitions/search.js:34]: This is subject to change when the content arrives.

File: test/browser/features/translation.feature [hunk 2]
```diff
@@ -0,0 +1,20 @@
+@mock-api:address-success @success
+Feature: Happy Path - changing languages
+  Change languages based on cookie
+
+  Background:
+    Given Authenticalable Address Amy is using the system
```
Comment (jkunle) [inline test/browser/features/translation.feature:6]: Maybe `Authenticable` or `Authenticatable`

Comment (MattBidewell) [reply test/browser/features/translation.feature:6]: I can raise this in a separate PR as that would require a change to all 13 test.feature files

**Review by idrop — APPROVED** [review]

**Pull request merged**